suites/pacific/rgw/tier-2_rgw_regression.yaml

```diff
@@ -100,36 +100,36 @@ tests:
       name: deploy cluster
       polarion-id: CEPH-83573713
 
-  - test:
-      name: Monitoring Services deployment
-      desc: Add monitoring services using spec file.
-      module: test_cephadm.py
-      polarion-id: CEPH-83574727
-      config:
-        steps:
-          - config:
-              command: apply_spec
-              service: orch
-              validate-spec-services: true
-              specs:
-                - service_type: prometheus
-                  placement:
-                    count: 1
-                    nodes:
-                      - node1
-                - service_type: grafana
-                  placement:
-                    nodes:
-                      - node1
-                - service_type: alertmanager
-                  placement:
-                    count: 1
-                - service_type: node-exporter
-                  placement:
-                    host_pattern: "*"
-                - service_type: crash
-                  placement:
-                    host_pattern: "*"
+  #- test:
```

Comment:
Any reason why monitoring service deployment test-case is commented?
please remove the comment if it is not intended.